test/e2e/networkingnew.go

```diff
@@ -0,0 +1,358 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
FYI I've asked Jay to move this code back into networking.go instead of having 2 copies of a networking test that are essentially identical in what they test.